x-pack/plugins/siem/public/containers/source/index.tsx

```diff
@@ -64,6 +64,7 @@ export class WithSource extends React.PureComponent<WithSourceProps> {
           const logAlias = get('source.configuration.logAlias', data);
           const auditbeatAlias = get('source.configuration.auditbeatAlias', data);
           const packetbeatAlias = get('source.configuration.packetbeatAlias', data);
+          const winbeatAlias = get('source.configuration.winbeatAlias', data);
```

Comment:
Looks like you missed adding to the `WithSourceArgs` up above. `Packetbeat` is missing as well, so I'd be fine with just opening a ticket to clean up how this component works.

Reply:
Good eyes. I just fixed that now. Also missed adding it to the ANY above and fixed that as well.